lib/content/audio/following_train.ex

```diff
@@ -60,36 +60,51 @@ defmodule Content.Audio.FollowingTrain do
     def to_params(audio) do
       case Utilities.destination_var(audio.destination) do
         {:ok, dest_var} ->
-          green_line_branch = Content.Utilities.route_branch_letter(audio.route_id)
-
-          cond do
-            !is_nil(green_line_branch) ->
-              green_line_with_branch_params(audio, green_line_branch, dest_var)
-
-            audio.minutes == 1 ->
-              {:canned, {"159", [dest_var, verb_var(audio)], :audio}}
-
-            true ->
-              {:canned, {"160", [dest_var, verb_var(audio), minutes_var(audio)], :audio}}
+          if audio.destination in [
+               :southbound,
+               :northbound,
+               :westbound,
+               :eastbound,
+               :inbound,
+               :outbound
+             ] do
```

Comment:
Since we're doing this check multiple times, what about creating a helper, e.g. `directional_destination?`

Reply:
Yeah good point!